copier/main.py

```diff
@@ -590,6 +590,7 @@ def run_copy(self) -> None:
             print("")  # padding space
         src_abspath = self.template_copy_root
         try:
+            print(f"Copying from template version {self.template.commit}")
```

Comment:
Don't you think the version (after processing to the PEP440 equivalent) would be more helpful?

Besides, please print to STDERR:

```suggestion
            print(f"Copying from template version {self.template.version}", file=sys.stderr)
```

Reply:
Ah, I hadn't seen that attribute, it probably does make more sense. I'm not sure what that contains when copying from a modified commit, or from a vcs-ref like HEAD, I'll double check them myself.